### DISCUSSION TITLE
Unzipping multiple files in the same folder?

### DISCUSSION THREAD

**Discussion by IdolR** [aonez/Keka#1391] [Q&A] [answered]
Not sure why, but if multiple zips contain data from one original folder, unzipped files will be expanded in multiple folders "Take out", Takeout 2", "Takeout 3", etc.
How can I avoid this?

**Comment by aonez** [comment, marked as answer]
Sorry this is not currently doable. You need to merge them manually. Those files are probably from Google Drive or similar, I'm not sure why they split them in such a way.

**Reply by IdolR** [reply]
Sorry but I have requested a refund, for two reasons.

- firstly there's no preview of zip contents, before extracting data and eventually selecting individual files
- secondly, there should be an option to eaxpnd all content to the same folder. This is basic functionality, which for example is achieved by the command line unzip \*.zip command

**Reply by RenatoRolando** [reply]
I have the same problem: GogglePhotos spilt all my photos on 2Gb zip folders for download.
Each zip file have a Takeout root folder.
I have 72 zip files! is very annoying merge all manually!
